src/content/ui/assets/assets-and-images.md

```diff
@@ -60,6 +60,13 @@ During a build, Flutter places assets into a special
 archive called the _asset bundle_ that apps read
 from at runtime.
 
+### Automatic transformation of asset files at build time
+
+You can have asset files transformed by dart packages as they are bundled into
+your app.
+To learn how to do this and write your own asset-transforming packages, see
+[Transforming assets at build time][].
+
```

Comment:
I imagine this to be a fairly niche (even if highly useful) feature, so I didn't want to fully explain it on this page but rather give it its own page and link to that. However, this section feels a bit sparse (and too easy to gloss over) to me. If anyone has ideas to improve this, please let me know.